examples/example-1/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "example-1",
+  "version": "1.0.0",
+  "description": "example broccoli-eyeglass project 1",
+  "main": "Brocfile.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Alan Hogan",
+  "license": "SAX-PD",
```

Comment:
Sure, that’s fine. I figured there’s nothing more liberal than public domain, but I guess standard licenses limit liability.